contracts/Registrar.sol

```diff
@@ -147,7 +138,7 @@ contract Registrar {
 
     /// Set the owner of the domain.
     function setDomainOwner(address newOwner) public adminOnly {
-        ens.setOwner(namehash(parentNode, baseLabel), newOwner);
+        ens.setOwner(domain, newOwner);
```

Comment:
Should we be using `ens.transferFrom` actually? :thinking:

Reply:
I'm sure that `setOwner` is used here correctly. The registrar will be the owner of the `radicle.eth` domain and it's the only address entitled to pass it to someone else.

What does `transferFrom` do? The ENS docs only mention it once, in context of the ERC-721 tokens API.

Reply:
I asked one of the ENS people, and the answer was "`transferFrom` should be enough". :shrug:

Reply:
Fixed